diagnostic.md

```diff
@@ -16,13 +16,13 @@ Graces-MacBook-Pro:try grace$
 
 What is the full path to the current directory?
 
-_Replace this text with your answer._
+/Users/grace/wdi/try
 
 ## Question 2
 
 Describe GitHub, **briefly** in your own words.
 
-_Replace this text with your answer._
+GitHub is a place where you can push your code into a repository. This makes it so you can track edits you have made to code over time as well as share code with others.
```

Comment:
FYI, you may want to Google the term "version control" - it's the more technical term for what you are describing.